deny.toml

```diff
@@ -18,7 +18,7 @@
 # dependencies not shared by any other crates, would be ignored, as the target
 # list here is effectively saying which targets you are building for.
 targets = [
-    # The triple can be any string, but only the target triples built in to
```

Comment:
Looks like this comes from https://github.com/EmbarkStudios/cargo-deny/blob/1188b6d053037c4b7d998bc002d97b603ff7ebc3/deny.template.toml#L21, in case you care enough to send them a PR as well

Reply:
Added to https://github.com/stars/jsoref/lists/to-check-spelling (it's a long list...)